docs/DeveloperGuide.md

```diff
@@ -81,13 +81,16 @@ The architecture design is given in the diagram above. The main components of No
 
 #### <a id="notus"><ins>2.2 NotUS</ins></a>
 
-NotUS manages the flow of the application. On launch, it will create the necessary components, as listed above and then attempts to load any existing saved files into the application. Subsequently, it will accept and interpret the user input and execute the commands accordingly. The diagram below depicts the main flow of the application.
+NotUS manages the flow of the application. On launch, it will create the necessary components, as listed above and then attempts to load any existing saved files into the application. Subsequently, it will accept and interpret the user input and execute the commands accordingly. Figure 2 below depicts the main flow of the application.
 
 <p align="center">
   <img alt="NotUS" src="diagrams/out/Notus.png" />
   <br><em>Figure 2</em>
 </p>
 
+💡 Due to the limitation of 
+💡 The lifeline for Parser and Command should end at destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram. This applies to the rest of the sequence diagrams in the document.
```

Comment:
Can verify this again? Looks off.

Reply:
https://nus-cs2113-ay2021s1.github.io/website/schedule/week10/topics.html#w10-2-sequence-diagrams-intermediate-level

Reply:
Not the content haha, line 91 just ends with "due to the limitation of".

Reply:
Noted.